addons/stock/views/product_views.xml

```diff
@@ -174,6 +174,7 @@
             <field name="inherit_id" ref="product.product_variant_easy_edit_view"/>
             <field name="arch" type="xml">
                 <sheet position="before">
+                    <field name="nbr_reordering_rules" invisible="1"/>
```

Comment:
this isn't ok, you should be able to replenish in an mto/buy scenario

Reply:
Changed this to be based on the routes instead (I misunderstood what a replenishment is) via a `replenish_visible` boolean. I attempted to recreate the same logic used by `product_replenish.py:default_get()` for `warehouse_id` and `location_id` values to pass into `stock_rule.py:_get_rule()` to see if any replenish rules exist (i.e. I tried to check for when the replenish button would successfully start a replenishment using `stock_rule.py:run()`, but ignored the cases when other things are misconfigured.)